### ISSUE TITLE
Private setting of a different remote.

### ISSUE THREAD

**Issue by rainwoodman** [PersonifyInc/git-fat#8]
This would require moving .gitfat into .git/fatconfig or .git/config.

Which is sensible because the S3 backend would prefer not to distribute .gitfat within the repository anyways. #7

**Comment by dlin** [comment]
A possible workaround (which is what I do) is to use environment variables to specify my keys: https://github.com/PersonifyInc/git-fat/commit/8a9775f8bdce69d5776fd906c17d37e0fe8d6080

I guess I should have documented that feature better.

**Comment by rainwoodman** [comment]
shall we merge this to #7

**Comment by zelonght** [comment]
Re: "A possible workaround (which is what I do) is to use environment variables to specify my keys: 8a9775f" -- for S3 use key as Environment params, for rsync, use .gitfat, not a bad idea at all.

I don't use rsync, so let's @dlin and @willkelleher decide on this.

**Comment by zelonght** [comment]
re: "shall we merge this to #7" -- wait a bit, if we go as @dlin suggest, so #7 is likely not necessary, right?

**Comment by rainwoodman** [comment]
right. environment variables make sense to me too, after some thoughts. So this document really shall mention the use of s3 key environment too. And we may want to patch the code to warn about s3 credentials in .gitfat files; even filter it out by default from a commit...
